usb/drv.c

```diff
@@ -357,13 +360,15 @@ int usb_drvUnbind(usb_drvpriv_t *drv, usb_dev_t *dev, int iface)
 }
 
 
-int usb_drvBind(usb_dev_t *dev)
+int usb_drvBind(usb_dev_t *dev, usb_drvOnBindCb_t onBindCb)
 {
 	usb_drvpriv_t *drv;
 
 	msg_t msg = { 0 };
```

Comment:
Note that `msg` is initialized only once, but it's being sent in a loop. Only the first `msgSend` transmits the original data - subsequent calls send the modified version. You need to reset `msg` before each `msgSend`.